log4j-core/src/main/java/org/apache/logging/log4j/core/impl/ThreadContextDataInjector.java

```diff
@@ -87,19 +87,33 @@ private static List<ContextDataProvider> getServiceProviders() {
         return Collections.unmodifiableList(providers);
     }
 
+    private abstract static class AbstractContextDataInjector implements ContextDataInjector {
+
+        protected final List<ContextDataProvider> providers;
+
+        protected AbstractContextDataInjector() {
```

Comment:
Can't these two `protected` be `private` instead?

Reply:
I think this is mostly a question of taste, since the class is `private`.

I switched them to package private in https://github.com/apache/logging-log4j2/pull/2846/commits/c475dbf0af351bc30dc7f94e3f943ce4386ca0fa, so we don't need any synthetic accessors.